unidock_tools/Dockerfile

```diff
@@ -27,5 +27,5 @@ RUN pip install tqdm
 
 COPY . /opt/unidock_tools
 RUN cd /opt/unidock_tools && \
-    pip install . && \
+    pip install . --no-deps && \
```

Comment:
Why no-deps here?

Reply:
All dependencies have been installed by conda in previous build stages, there is no need to install them again.

Reply:
I would suggest use dependencies solved in pip, but not installing them ahead in conda manually

Reply:
> I would suggest use dependencies solved in pip, but not installing them ahead in conda manually

Done. Dependencies will be installed by pip if they are in pypi.